sdk/forms/src/protected-view/iframe/components/view.ts

```diff
@@ -44,49 +65,46 @@ const DisplayValue = component(
         span.innerText = String(value);
         container.appendChild(span);
         children.push(span);
```

Comment:
Remove `container.appendChild(span);` line.
Calling `children.push(span);` is sufficient here because we later call `container.replaceChildren(...children);`.

Reply:
Agree, removed redundant appendChild